widgetssdk/src/main/java/com/glia/widgets/callvisualizer/controller/CallVisualizerController.kt

```diff
@@ -43,12 +43,12 @@ internal class CallVisualizerController(
 
     override fun onOneWayMediaUpgradeRequest(mediaUpgradeOffer: MediaUpgradeOffer, operatorName: String) {
         val formattedOperatorName = Utils.formatOperatorName(operatorName)
-        dialogController.showUpgradeVideoDialog2Way(mediaUpgradeOffer, formattedOperatorName)
+        dialogController.showUpgradeVideoDialog1Way(mediaUpgradeOffer, formattedOperatorName)
```

Comment:
Wow! So it was working vice versa until now?

Reply:
Yes, that's good that @gugalo has noticed that. 👍 Thank you.